packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -234,6 +268,16 @@ const PayableDetailsFormBase = forwardRef<
 
     const className = 'Monite-PayableDetailsForm';
 
+    const { ocrRequiredFields } = useOcrFields(inProps);
+
+    useEffect(() => {
+      const isRequiredFieldMissed = ocrRequiredFields
+        ? Object.values(ocrRequiredFields).some((required) => required)
+        : false;
+
+      if (isRequiredFieldMissed) trigger();
```

Comment:
I would simplify the logic as `ocrFieldsToTrigger.forEach(field => trigger('field', field)`